torch/csrc/utils/tensor_memoryformats.cpp

```diff
@@ -12,26 +12,37 @@
 namespace torch {
 namespace utils {
 
-#define _ADD_MEMORY_FORMAT(format, name)                                       \
-  {                                                                            \
-    std::string module_name = "torch.";                                        \
-    PyObject* memory_format = THPMemoryFormat_New(format, module_name + name); \
-    Py_INCREF(memory_format);                                                  \
-    if (PyModule_AddObject(torch_module, name, memory_format) != 0) {          \
-      throw python_error();                                                    \
-    }                                                                          \
-  }
+namespace {
+// Intentionally leaked
```

Comment:
Not saying we should, but out of curiosity: could we store unique_ptr with custom deleter in this array to avoid leaking these?

Reply:
The point of leaking them is to avoid C++ / CPython unloading ordreing disasters